client/src/components/reader/ReturnMenuButton.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component, PropTypes } from 'react';
+
+export default class AppearanceMenuButton extends Component {
+
+  static propTypes = {
+    toggleReaderMenu: PropTypes.func,
```

Comment:
Because this component would throw an exception if toggleReaderMenu were not provided, it should be marked as required (or there should be a noop function as a defaultProp, if you prefer)